src/core/style/_elevation.scss

```diff
@@ -0,0 +1,186 @@
+/**
+ * A collection of mixins and CSS classes that can be used to apply elevation to a material
+ * element.
+ * See: https://www.google.com/design/spec/what-is-material/elevation-shadows.html
+ * Examples:
+ *
+ * ```scss
+ * .md-foo {
+ *   @include $md-elevation(2);
+ *
+ *   &:active {
+ *     @include $md-elevation(8);
+ *   }
+ * }
+ * ```
+ *
+ * ```html
+ * <div id="external-card" class="md-elevation-z2"><p>Some content</p></div>
```

Comment:
What will happen, when `ViewEncapsulation` is disabled and you import the `elevation` styles in those components. Won't that cause some duplicates? 

Or in general, how can the user access these classes? Will there be a standalone stylesheet?